test/OntologyTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+namespace oat\taoLom\test;
+
+use oat\generis\model\OntologyAwareTrait;
+use oat\tao\test\TaoPhpUnitTestRunner;
+
+class OntologyTest extends TaoPhpUnitTestRunner
+{
+    use OntologyAwareTrait;
+    
+    public function testGeneralOntologyClass()
+    {
+        $categoryClass = $this->getClass('http://www.tao.lu/Ontologies/TAOLom.rdf#LomCategory');
```

Comment:
Could you please update your unit tests to reflect the new URI Scheme in use (taotesting.com instead of tao.lu) in the unit tests?